### ISSUE TITLE
End of support

### ISSUE THREAD

**Issue by Rudloff** [Rudloff/alltube#435]
TLDR: I will stop maintaining this repository.

The main reason is I don't really use this software myself anymore.

Since #422, there is no easy way to use AllTube so people try to set it up themselves or use the (very outdated) Docker image.
I find that helping people debug their setup or maintaining a Docker I don't use myself is not very rewarding.

I have also been getting regular demands from users that are trying to profit from the software (like hosting a version with ads) and asking me for help without even bothering to read the contribution guidelines. This is not something I want to help people with.

I will probably publish one last release with the latest fixes (mainly the switch to yt-dlp) and then make the repository read-only.

The code will remain available so other people are of course still free to use it and build upon it.
There is also [a library](https://github.com/Rudloff/alltube-library) containing the core functionalities that people can use to build their own GUI.

Thanks to the people that contributed to this repository or reported bugs over the years!

**Comment by mariushosting** [comment]
Thank you for the update and I hope that the last release will solve the yt-dlp issue. I share your opinion and you are right.

**Comment by rosano** [comment]
Sorry to hear this. I have managed to self-host via Cloudron (which seems to be using yt-dlp already) and there have never been issues.

**Comment by Rudloff** [comment]
Release containing the latest changes has been published: https://github.com/Rudloff/alltube/releases/tag/3.2.0-alpha
